packages/core/__test__/elements/FormFieldElement.test.js

```diff
@@ -188,7 +188,7 @@ describe('elements/FormFieldElement/SelectionTest', () => {
   it('Applies `selectionDirection`', async () => {
     const formFieldEl = await fixture('<form-field-element-test></form-field-element-test>');
     const inputElement = formFieldEl.inputElement;
-    const selectionDirection = 'backward';
+    const selectionDirection = 'forward';
```

Comment:
This changes the test case initial intention. Why does it need to be `forward` ?

Reply:
This is the bug that I was mentioned in the previous meeting. I am not sure what ish the root cause, In this test case when the selection direction is `backward` it cannot set/change to the new value. I will create bug in JIRA.

Reply:
Please leave a reason on changed code.

Reply:
To make the testing on Safari pass

Reply:
I meant a comment on the code itself.

Reply:
Added